docs/demos/price_oracle_intro.md

```diff
@@ -28,86 +27,41 @@ This demo is composed of:
 node.
 
 <figure markdown>
-![](./images/oracle_diagram.svg)
+![](../images/oracle_diagram.svg)
 <figcaption>Architecture of the Price Oracle demo</figcaption>
 </figure>
 
 
-## Running the Demo
+## Running the demo
 The steps below will guide you to download the price oracle agent service definition from the Service Registry, build and run a deployment locally.
 
-1. Ensure that your machine satisfies the [framework requirements](guides/quick_start.md#requirements) and that
-you have followed the [setup instructions](guides/quick_start.md#setup). As a result you should have a Pipenv workspace folder.
-
-2. Use the CLI to download the `valory/oracle_hardhat` service.
+1. Open a terminal and run a local [HardHat](https://hardhat.org/) node that will emulate a blockchain node. For convenience, we provide a Docker image in [Docker Hub](https://hub.docker.com/) that can be run by executing:
     ```bash
-    autonomy fetch valory/oracle_hardhat:0.1.0:bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe --remote --service
-    cd oracle_hardhat
-    ```
-
-3. Prepare a JSON file `keys.json` containing the addresses and keys of the four agents that are
```

Comment:
Where is this now?

Reply:
As you can see, the instructions guides have been simplified (this is less error-prone), and now the guides reference "macro-steps":. This is now subsumed in the "deploy local service guide", and is referenced in line 54 (right side):

```3. Follow the steps in the [local deployment guide](../guides/deploy_service.md#local-deployment) to deploy and run the service locally.```